examples/prelude/README.md

```diff
@@ -17,9 +17,21 @@ ln -s $(realpath ../../prelude) prelude
 
 If you are using any other prelude directory, make sure the prelude points to it in `.buckconfig`
 
+On Linux and macOS, an alternative to symlinking the prelude manually in this way is to `source ./setup.sh` (see the next section for details).
+
+## Support for building the example OCaml targets
+
+The information in this section is (at this time) Linux and macOS specific.
+
+The commands in 'setup.sh' assume an [opam](https://opam.ocaml.org/) installation.
```

Comment:
Usually you would do backticks to make it teletype in markdown

Reply:
ok